src/utils/chatService.js

```diff
@@ -68,7 +69,9 @@ export default class ChatService {
 
   onNewListenAddr = () => {
     this.event.addListener('newListenAddr', ((multiaddr) => {
-      console.log(multiaddr);
+      if (multiaddr.includes('192')) { // this avoid passing local address to the state
```

Comment:
```suggestion
      if (multiaddr.startsWith('/ip4/192.168') ||
          multiaddr.startsWith('/ip4/10') ||
          multiaddr.startsWith('/ip4/172.16')) { // this avoid passing local address to the state
```

This to cover edge cases in case someone modifies android (i'm looking at you chinese phones :eyes: ) to use a different kind of private addresses. Using `startsWith` prevents `/ip6/fc00::192a/` from being parsed incorrectly.

Reply:
this will be removed the user can select a list of addresses.

Reply:
this will be removed the user can select a list of addresses